pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
Much hardware is level interrupt only, and with some hardware it is the only sane way to deal with it. I find level interrupts to be much more reliable, and less painful. edge stuff, when it is comming in too fast, can give you a really bad day, and the MAX3421E is an excellent example of a chip that can outrun many processors.